Security Recommendations/Private preview recommendations/TVM-recommendations/SecurityPolicies/TVM-EnableVTPM-Audit/azurepolicy.rules.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
why do we have separate file for rules only?

Reply:
I am not sure, but this is how the other policies are maintained in this repository - policy, rules and parameters